qiita_db/parameters.py

```diff
@@ -172,6 +172,27 @@ def to_str(self):
 
         return " ".join(result)
 
+    @classmethod
+    def get_all_parameters(self):
```

Comment:
NABI: This name seems way too general, but I don't have suggestions to improve it.

------
NABI = not a blocking issue.